ui/src/main/js/field/MessageFormatter.js

```diff
@@ -53,11 +52,13 @@ class MessageFormatter extends Component {
 
 MessageFormatter.propTypes = {
     id: PropTypes.string,
+    isDetailed: PropTypes.bool,
     message: PropTypes.string
 };
 
 MessageFormatter.defaultProps = {
     id: 'messageFormatterId',
+    isDetailed: false,
```

Comment:
Might be nice to use the same `errorIsDetailed` variable name here as in `GlobalConfiguration` so it's obvious that they represent the same thing.